robotpy_ext/control/toggle.py

```diff
@@ -0,0 +1,107 @@
+from _functools import partial
+
+
+class Toggle:
+    """Utility class for button toggle
+
+        Usage:
+
+        foo = Toggle(joystick, 3)
+
+        if foo:
+            toggleFunction()
+
+        if foo.on:
+            onToggle()
+
+        if foo.off:
+            offToggle()
+    """
+    class _SteadyDebounce:
```

Comment:
It's as useful as a normal debounce I guess? If you think you need it, you can put it in, if you don't, leave it out.